spec/validation_spec.js

```diff
@@ -1,42 +1,52 @@
 var htmlbook = require('../htmlbook');
 var fs = require('fs');
 var exec = require('child_process').exec;
+const tmp = require('tmp');
 
-jasmine.getEnv().defaultTimeoutInterval = 100000;
+tmp.setGracefulCleanup();
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000;
 
 var convert_and_validate = function (file_name, callback) {
-  fs.readFile(file_name, "utf-8", function (err, source) {
+  tmp.file((err, path, _) => {
+    if (err) throw err;
+    fs.readFile(file_name, "utf-8", function (err, source) {
+      if (err) throw err;
 
-    html =  htmlbook(source).parse({"complete_html": true, "title": "Test"})
+      html = htmlbook(source).parse({ "complete_html": true, "title": "Test" })
 
-    fs.writeFile("spec/documents/validation.html", html, function () {
-      exec("xmllint --noout --schema ../HTMLBook/schema/htmlbook.xsd spec/documents/validation.html", function (err, stdout, stderr) {
-        expect(stderr).toEqual("spec/documents/validation.html validates\n");
-        fs.unlink("spec/documents/validation.html");
-        callback();
-      });
-    })
-  });
+      fs.writeFile(`${path}.html`, html, function () {
+        exec(`xmllint --noout --schema ../HTMLBook/schema/htmlbook.xsd ${path}.html`, function (err, _stdout, stderr) {
+          if (err) throw err;
+          expect(stderr).toEqual(`${path}.html validates\n`);
+          callback();
+        });
+      })
+    });
+
+  }
+
+  )
 }
 
 describe("htmlbook validations", function () {
-  it("BBEPart2", function (done) {
+  xit("BBEPart2", function (done) {
```

Comment:
Do we need to keep these tests if they're no longer needed? Or figure out how to fix at some point?

Reply:
These have disappeared from the repo; I'd like to try and find them eventually, but maybe it's not worth it? I could go either way; this is not a heavily used part of Atlas (and in fact we recommend against it, strongly).